sklego/linear_model.py

```diff
@@ -64,3 +73,126 @@ def predict(self, X):
         X = check_array(X, estimator=self, dtype=FLOAT_DTYPES)
         check_is_fitted(self, ['coefs_'])
         return np.dot(X, self.coefs_)
+
+
+class LoessRegressor(BaseEstimator, RegressorMixin):
+    """
```

Comment:
this description can be moved to the `__init__` with an explainer of all the settings